index.html

```diff
@@ -18,6 +18,7 @@ <h2>Score</h2>
     </div>
     <div id="memory-board"></div>
 
-    <!-- LINK THE JAVASCRIPT FILES HERE (keep in mind that the order in which you link them MATTERS) -->
+      <script src="js/memory.js"></script>
+    <script src="js/index.js"></script>
   </body>
 </html>
```

Comment:
Great stuff Roy-Arne!
Please checkout my comments for potential improvements.
Please reach out if you have any question.

Reply:
.